src/Encryption.php

```
@@ -366,14 +367,14 @@ private static function convertBase64ToBigInteger(string $value): BigInteger
     {
         $value = unpack('H*', Base64Url::decode($value));
 
-        return BigInteger::fromBase($value[1], 16);
+        return BigInteger::fromBase($value !== false ? $value[1] : '', 16);
     }
 
     private static function convertBase64ToGMP(string $value): \GMP
     {
         $value = unpack('H*', Base64Url::decode($value));
 
-        return gmp_init($value[1], 16);
+        return gmp_init($value !== false ? $value[1] : '', 16);
```

Maybe instead throw a readable error if unpacking has failed?

---

Added an exception.